WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -29,7 +29,7 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
     @Published private(set) var description: String = ""
 
     // Budget details
-    private var startDate = Date.now
+    private var startDate = Date.now + 60 * 60 * 24 // Current date + 1 day
```

Comment:
Just curious about this. Is it a common pattern in iOS to work with seconds for time, instead of millis?